docs/v2/sameSiteCookies.md

```diff
@@ -1,6 +1,6 @@
 ## Description
 ```php
-void sameSiteCookies ([ string $mode = null ] )
+void sameSiteCookies ([ ?string $mode = null ] )
```

Comment:
For visibility, the only docs change was the above instance where `$mode` was nullable, has a `null` default, but wasn't explicitly marked as nullable.